Makefile

```diff
@@ -46,6 +46,22 @@ format:
 	./mvnw compile test-compile scalafix:scalafix -Psemanticdb $(PROFILES)
 	./mvnw spotless:apply $(PROFILES)
 
+# build native libs for amd64 architecture Linux/MacOS on a Linux/amd64 machine/container
+core-amd64-libs:
+	cd native && cargo build -j 2 --release
+ifdef HAS_OSXCROSS
+	rustup target add x86_64-apple-darwin
+	cd native && cargo build -j 2 --target x86_64-apple-darwin --release
+endif
```

Comment:
So as the L51 is not in an `else` block, if `HAS_OSXCROSS` is true, we will build the library for x86_64-apple-darwin additionally? I.e., two libraries for `core-amd64-libs`?

Reply:
Yes.  For the amd64 architecture, one for linux and one for MacOS